case-studies/eshoppen

```diff
@@ -0,0 +1,156 @@
+* No GPU was used hence Pg ~0
+* TDP of server used in Azure App server Premium configuration (P2v2 ) -2nd Generation Intel® Xeon® Platinum 8272CL (Cascade Lake)= 205 W ( [https://ark.intel.com/content/www/us/en/ark/products/192482/intel-xeon-platinum-8270-processor-35-75m-cache-2-70-ghz.html](https://ark.intel.com/content/www/us/en/ark/products/192482/intel-xeon-platinum-8270-processor-35-75m-cache-2-70-ghz.html) )
+
+### Carbon Intensity (`I`)
+
+* We will use regional yearly averages.
+* The region the application was run in was India.
+* We will source the Carbon Intensity from the SCI Data project and the [[I] Regional Yearly Marginal Carbon Intensity](https://docs.google.com/spreadsheets/d/1Viv94rMKH-fJrfD9Nn9_qkiAg1PDfIfJUAHRX9slG7A/edit#gid=1582216595) data set.
+
+### Embodied Carbon (`M`)
+
+The equation to calculate `M = TE * (TR/EL) * (RR/TR)`
+
+Where:
+
+* TE = Total Embodied Emissions, the sum of LCA emissions for all hardware components.
+* TR = Time Reserved, the length of time the hardware is reserved for use by the software.
+* EL = Expected Lifespan, the anticipated time that the equipment will be installed.
+* RR = Resources Reserved, the number of resources reserved for use by the software.
+* TR = Total Resources, the total number of resources available.
+
+For this component:
+
+* TE: We will source the embodied carbon estimates for the servers from the [Cloud Carbon Footprint Coefficient](https://github.com/cloud-carbon-footprint/cloud-carbon-coefficients/blob/main/output/coefficients-azure-embodied.csv) Data Set.
+* TR: 1 hr.
+* EL: We will assume a 4 year lifespan or 35,040 hrs.
+* RR: A virtual machine with 2 vCPUs was used, this data was sourced from [Cloud Carbon Footprint Azure Instances Coefficients](https://github.com/cloud-carbon-footprint/cloud-carbon-coefficients/blob/main/data/azure-instances.csv).
+* TR: The bare metal host is split up into 16 vCPUs in total. This data was sourced from the [Cloud Carbon Footprint Azure Instances Coefficients](https://github.com/cloud-carbon-footprint/cloud-carbon-coefficients/blob/main/data/azure-instances.csv).
+
+## (Quantify) SCI Value Calculation
+
+*Show your work! For each of the components of your software system, show how you arrived at the SCI value. Guidance for this is available in the [Methodology summary](https://github.com/Green-Software-Foundation/software_carbon_intensity/blob/main/Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md#methodology-summary) section.
+*
+
+### App server for Web application
+
+#### Energy (`E`)
+
+_The workings of E, include raw numbers and calculations._
+
+- Server utilization = 18.3922%
+- Number of hours = 1
+- Number of cores = 2 
+- TDP = 205W
+- TDP Coefficient = 0.32
+
+```
+E = Server utilization * Number of hours * Number of cores * TDP * TDP co-efficient
```

Comment:
It should not use cores for this formula - TDP is affected to the chip as I pointed in #250 .

Reply:
@YaSuenag  In the latest commit I have fixed the above comment. Is this still not visible iin the pull request ? https://github.com/Green-Software-Foundation/software_carbon_intensity/pull/256/commits/b92bcb27f18ace87bc2b5d3471dbb2ba3f524ac7

Reply:
I can see https://github.com/Green-Software-Foundation/software_carbon_intensity/commit/b92bcb27f18ace87bc2b5d3471dbb2ba3f524ac7 , however it does not seem to belong to any branch on SCI repository.

![screenshot](https://user-images.githubusercontent.com/7421132/174503974-d6009b16-441e-4ae6-bf8b-fe3a7dc91352.png)

I think this PR attempts to merge dev branch on SCI, and it will contain incorrect formula if it has done. So I think you should fix the formula in this PR.

Reply:
There were 2 versions of the file and I deleted the incorrect one. @YaSuenag  Please look at the latest file attached to this PR